opening_hours/utils/hauki_exporter.py

```diff
@@ -14,128 +16,110 @@ class HaukiResource:
     id: int | None
     name: HaukiTranslatedField
     description: HaukiTranslatedField
-    address: HaukiTranslatedField | None
-    children: list[int]
     parents: list[int]
     organization: str
     origin_id: str
-    origin_data_source_name: str
-    origin_data_source_id: str
-    resource_type: ResourceType = ResourceType.RESERVABLE
 
-    def convert_to_request_data(self):
+
+class ReservationUnitHaukiExporter:
+    reservation_unit: ReservationUnit
+    hauki_api_client: HaukiAPIClient = HaukiAPIClient()
+
+    def __init__(self, reservation_unit: ReservationUnit):
+        self.reservation_unit = reservation_unit
+
+        if self.reservation_unit is None:
```

Comment:
`reservation_unit` typed as just `ReservationUnit` and not `ReservationUnit | None`, so it should be the caller's responsibility to filter the value before it is given if it could be `None`.